### ISSUE TITLE
shortcut for replies

### ISSUE THREAD

**Issue by lub** [vector-im/riot-web#10733]
It would be great to use replies without leaving the keyboard.
E.g. [ctrl]+[arrow key] to select a message and [return] to confirm

https://matrix.to/#/!YTvKGNlinIzlkMTVRl:matrix.org/$YC3rGXMm3Q7yUgkUrg9RdcxT2---d_5-mFXkaTWUA8I?via=matrix.org&via=privacytools.io&via=tedomum.net

**Comment by Arctize** [comment]
Will this ever be considered?
Coming from Telegram Desktop, I'd really appreciate such a shortcut, and I know many others who would too.

**Comment by t3chguy** [comment]
Yes, that's why the issue is open. If it wasn't to be considered it would be closed.

**Comment by ara4n** [comment]
alt-up/down currently navigates you up and down the roomlist. alt-meta-up/down (i.e. alt-cmd-up/down on macOS) might work though?

**Comment by SimonBrandner** [comment]
> alt-meta-up/down

Isn't that a composer history shortcut?

**Comment by SimonBrandner** [comment]
[#17128](https://github.com/vector-im/element-web/issues/17128) is somewhat related as we want to avoid collisions

**Comment by ara4n** [comment]
> Isn't that a composer history shortcut?

on macOS at least the composer history shortcut is ctrl-alt-up/down, not alt-meta-up/down.

**Comment by SimonBrandner** [comment]
Sorry, I am getting confused by how keys are called in different places... 

![Screenshot_20210523_193622](https://user-images.githubusercontent.com/25768714/119270676-29110280-bbfe-11eb-8e8f-64025ac0eb4d.png)
![Screenshot_20210523_193632](https://user-images.githubusercontent.com/25768714/119270679-30381080-bbfe-11eb-8a56-51abebdb10b2.png)

This, at least on Windows and Linux, leaves shift + control. Unless we want to include the super/windows key (which I am not a fan of...). Sorry, if I am missing something here...

**Comment by t3chguy** [comment]
Shift + Control + Arrows is a text editing shortcut for selecting blocks, no?

**Comment by SimonBrandner** [comment]
Yep...

**Comment by Arctize** [comment]
Is ctrl-up / ctrl-down not possible? Or is that used/reserved for something else already?

**Comment by t3chguy** [comment]
Those are standard text field shortcuts

```

Ctrl + Down arrow | Move the cursor to the beginning of the next paragraph.
-- | --
Ctrl + Up arrow | Move the cursor to the beginning of the previous paragraph.

```

**Comment by Arctize** [comment]
Oh okay, thanks. I did not know that. Does this apply to other OSes or is this a Windows only thing?
I always assumed that since Telegram uses it, and since the shortcut doesn't currently do anything on Linux, it would still be available.

**Comment by SimonBrandner** [comment]
Looking into this now, the shortcut does a different thing in different places...

**Comment by c7hm4r** [comment]
I like Slack’s shortcuts. They allow to select any message with the up/down key (unless you are currently inputting a multi-line message :confused: ). Once a message is selected you can press letter keys without modifiers to perform actions on that message: `e` for edit, `t` for replying in a new threat, `r` for reactions.

Maybe it makes sense to consider this together with #16332.
